include/yaramod/builder/yara_rule_builder.h

```diff
@@ -26,6 +37,7 @@ class YaraRuleBuilder
 	/// @name Constructor
 	/// @{
 	YaraRuleBuilder();
+	YaraRuleBuilder(std::shared_ptr<TokenStream> tokenStream);
```

Comment:
No, that is unnecessary, because accepting shared_ptr parameter by value immediately increases the shared_ptr counter by one only to initialize the local variable used in the constructer. Then another copy is created from this local variable. One copying is definitely enough so I fixed it everywhere I found this problem. Thank you!